mgmt/MgmtDefs.h

```diff
@@ -72,71 +73,76 @@ struct MgmtConverter {
    * This is passed a @c void* which is a pointer to the member in the configuration instance.
    * This function must return a @c MgmtInt converted from that value.
    */
-  MgmtInt (*load_int)(const void *) = nullptr;
+  MgmtInt (*load_int)(const HttpSM *, const void *) = nullptr;
 
   /** Store a @c MgmtInt into a native type.
    *
    * This function is passed a @c void* which is a pointer to the member in the configuration
    * instance and a @c MgmtInt. The member should be updated to correspond to the @c MgmtInt value.
    */
-  void (*store_int)(void *, MgmtInt) = nullptr;
+  void (*store_int)(HttpSM *, void *, MgmtInt) = nullptr;
```

Comment:
Hmm the store puts the value in void *, so why would this change the state machine object?

Reply:
The original Idea was to store some data into the arena from the SM, which now will not take place here and should be in the `InkAPI` instead, this was the only reason why this wasn't `const`. After some chat with Alan last week, we agreed to take this (it will be on a diff PR) outside the scope of store_*,  so thanks for pointing this out. I'll adjust it.